packages/cli/test/Prompt.test.ts

```diff
@@ -37,5 +55,295 @@ describe("Prompt", () => {
 
         expect(result).toBe("default-value")
       }).pipe(runEffect))
+
+    it("should render the default value when the default is provided", () =>
+      Effect.gen(function*() {
+        const prompt = Prompt.text({
+          message: "Test Prompt",
+          default: "default-value"
+        })
+
+        const cli = Command.prompt("test-command", prompt, () => Effect.void).pipe(Command.run({
```

Comment:
Similar to the prior PR, I let's remove the CLI in favor of running the `Prompt` directly.

Reply:
How would I get the terminal lines out without creating a cli?

Reply:
Since MockTerminal uses Console you should still be able to use the same APIs

Reply:
I'll give it a try. There was a type error when I previously tried it, with just adding in the MockConsole but I just did what worked and moved on.